client/src/components/CartItems/CartItems.jsx

```diff
@@ -35,4 +35,11 @@ const mapStateToProps = (state) => {
     items: state.cartController.items
   }
 };
-export default connect(mapStateToProps)(CartItems);
\ No newline at end of file
+
+const mapDispatchToProps = (dispatch) => {
```

Comment:
Also, take a look at redux's `bindActionCreators`

Reply:
According to redux docs(quote you can see below), it doesn't give me any benefits in such case, but, anyway, thanks you for hint :). QUOTE: "The only use case for bindActionCreators is when you want to pass some action creators down to a component that isn't aware of Redux, and you don't want to pass dispatch or the Redux store to it." https://redux.js.org/api/bindactioncreators.